libs/@local/hash-isomorphic-utils/src/system-types/actor.ts

```diff
@@ -3,15 +3,19 @@
  */
 
 import {
-  Actor,
-  ActorOutgoingLinkAndTarget,
-  ActorOutgoingLinksByLinkEntityTypeId,
-  ActorProperties,
+  ActorV2,
```

Comment:
Any version qualifier in these generated types is an indication that an older version is still referenced by some existing system types – it's worth reviewing the generated types for these and figuring out what is referencing them still. In this case there's still a User v3 in the system types, which is probably why ActorV1 is still around. And User v3 is probably still around because there are other types which link to it – `comment`, `commentNotification`, `linearIntegration` and `mentionNotification` at least

One way of chasing these old references down is to use the GraphQL playground to get all entity types, and then search the old Verisoned URL in the output
```graphql
{
  queryEntityTypes(
    constrainsValuesOn: { outgoing: 0 }
    inheritsFrom: { outgoing: 0 }
    constrainsPropertiesOn: { outgoing: 0 }
    constrainsLinksOn: { outgoing: 0 }
    constrainsLinkDestinationsOn: { outgoing: 0 }
  ) {
    vertices
  }
}
```

Reply:
Thank you for spotting this, have addressed in https://github.com/hashintel/hash/pull/4031/commits/c0a1b842a20bc3091f01be292b66e08d00e2109c. I didn't realise the reason the version identifier is introduced is when there are more than one version in the system types output, but this makes sense to me know and will watch out for it in future. Thanks!